docs/src/pages/[platform]/connected-components/storage/storagemanager/props.ts

```diff
@@ -20,9 +20,14 @@ const eventHandler = `(file: {key: string}) => void;`;
 
 export const STORAGE_MANAGER = [
   {
-    name: `accessLevel`,
+    name: `path`,
+    description: 'Path in s3 to put the file under',
+    type: `string | (input: { identityId: string | undefined }) => string`,
+  },
+  {
+    name: `accessLevel?`,
     description:
-      'Access level for files in Storage. See https://docs.amplify.aws/lib/storage/configureaccess/q/platform/js/',
+      'Deprecated, use `path` instead. Access level for files in Storage. See https://docs.amplify.aws/javascript/build-a-backend/storage/configure-access/',
```

Comment:
There is some nuance with path and accessLevel that needs to be documented here. path is optional and only allows a string when accessLevel is provided, and required (and also allowed as a callback) when accessLevel is undefined